---
title: Research in our Lab
layout: default
group: research
---

<div class="row">

# Our research

The Hong Kong Polytechnic University Jockey Club STEM Lab of Genomics in Healthcare is dedicated to transforming biomedical research through the seamless integration of artificial intelligence, genomics, and experimental biology. We combine interdisciplinary expertise in AI, data science, genomics, molecular biology, and neuroscience. The research projects in the Lab are funded by over HKD 80 million from:
- **RGC** - Research Grants Council, University Grants Committee, Hong Kong
- **HMRF** - Health and Medical Research Fund, Health Bureau, Hong Kong
- **NSFC** - National Natural Science Foundation of China
- Hong Kong Government’s Global Talent Recruitment Scheme
- Hong Kong Jockey Club
<br>
<br>

</div>

<div class="row">

### Biology, Genetics and Genomics

<div class="col-md-12">

The work in the Zhang lab in this broad field is mainly computational and falls into the category of Data Sciences (or Big Data). We aim to find and understand biological “stories” through building models and identifying patterns from large quantities of genome-scale biological data. In the process, we integrate the bench work of molecular biology and develop novel methods and efficient software tools along the way. In this broad area, we pursue two overarching themes, **<span style="color:blue; font-weight:bold; font-style:italic;">network systems biology</span>** and **<span style="color:blue; font-weight:bold; font-style:italic;">gene regulation via noncoding RNA genes</span>**. We are interested in basic biological questions (e.g., the biogenesis of small noncoding RNAs, transcriptional and post-transcriptional gene regulation) and their applications to complex diseases (e.g., Alzheimer’s disease (AD) and psoriasis) and plant stress response (e.g., rice blast and rice bacterial blight).
- Network-based transcriptome analysis
- Biogenesis of small noncoding RNAs
- Regulatory functions of small noncoding RNAs

<img class="img-fluid" src="/static/img/research/research_picture1.png" alt="qFit">

</div>

<div class="row">

### Network GWAS

<div class="col-md-12">

In recent years, in order to support the research activities in computational biology and genomics, we have been focusing on developing effective computational methods for finding and analyzing modular structures in **<span style="color:blue; font-weight:bold; font-style:italic;">large (biological) networks</span>**.

The network GWAS and co-expression network approaches described above rely heavily on accurately and efficiently finding network modules. Nevertheless, finding structures in large networks is a challenging problem in machine learning and datamining. First, network structural properties can be characterized in several different ways, by node connectivity, by properties on links (such as relationships among connected individuals), or by semantics of nodes and links (such as the social roles that individuals play in a society network). In our research, we considered these different types of information for network module finding and developed effective novel methods for finding modules of nodes (Jin, Chen, He and Zhang, Proc. AAAI-15), modules of links (He, Liu, Jin and Zhang, Proc. AAAI-15), and modules that are defined by network structural information and node semantics (He, et al., Proc AAAI-17). We adopted a few different machine learning techniques in developing our methods, including non-negative matrix factorization (Wang, et al., Proc. AAAI-16), stochastic modeling (He, Liu, Jin and Zhang, Proc. AAAI-15), deep learning (Yang, et al., Proc. IJCAI-16), and Markov Random Fields (He, et al., Proc. AAAI-18).
- Network clustering analysis
- Identification of network modules in complex networks
- Applications of network module analysis

<img class="img-fluid w-50" src="/static/img/research/research_picture3.png" alt="antibiotic">

</div>

<div class="row">

### Genomic LLM

<div class="col-md-12">

We are currently developing **<span style="color:blue; font-weight:bold; font-style:italic;">deep learning</span>** algorithms and **<span style="color:blue; font-weight:bold; font-style:italic;">genomic foundational models</span>** for application in biomedical research. Our focus is on addressing critical challenges in genetics and molecular biology, such as noncoding gene regulation and genome-wide association studies. In our clinical research initiatives, we further harness these AI methodologies—particularly large language models—to automatically generate treatment strategies based on refined disease classifications and molecular markers.
- Genomic Foundation Models
- Specific models regarding different genomic problems

<img class="img-fluid w-75" src="/static/img/research/dnagrinder.png" alt="hairball">

</div>

<div class="row">

### Dedication

<div class="col-md-12">
  
Our laboratory is at the forefront of merging artificial intelligence with biomedical technology to address some of the most pressing challenges in modern medicine. By integrating **<span style="color:blue; font-weight:bold; font-style:italic;">computational innovation</span>** with rigorous **<span style="color:blue; font-weight:bold; font-style:italic;">experimental validation</span>**, we aim not only to advance our understanding of disease mechanisms but also to drive the development of personalized diagnostic and therapeutic strategies. Our early successes in psychiatric disorders and cancer underscore the potential of our interdisciplinary approach to yield transformative insights and clinical breakthroughs.


</div>
