<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"><script language="javascript" src="/module/jslib/jquery/jquery.js"></script>

<meta name='WebId' content='161'>

<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<title>Hong Kong Polytechnic University Research</title>
<meta name='Maketime' content='2023-04-13 19:42:47'>
<meta name="SiteName" content="Hong Kong Polytechnic University">
<meta name="SiteDomain" content="https://genomicmedicine.github.io/site/">
<meta name="SiteIDCode" content="">
<meta name="ColumnName" content="Research">
<meta name="ColumnDescription" content="">
<meta name="ColumnKeywords" content="">
<meta name="ColumnType" content="">
<meta name="channel" content="Research">
<meta name="chnlcatalog" content="">
<meta name="server" content="/site/col/col48302/index.html">
<meta name="i_webid" content="161">
<meta name="i_columnid" content="48302">
<meta name="i_articleid" content="0">
<meta name="pagetype" content="2">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="stylesheet" href="/site/images/6964/font-awesome.min.css">
<link href="/site/script/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="/site/images/6964/encommon.css">
<!--[if lt IE 9]>
<script src="https://cdn.bootcss.com/html5shiv/3.7.3/html5shiv.js"></script>
<script src="https://cdn.bootcss.com/respond.js/1.4.2/respond.min.js"></script>
<script>
        $(function () {
            $("#old-ie").css("display", "block");
        });
    </script>

<![endif]-->
<script src="/site/script/jquery.min.js"></script>
<script src="/site/script/bootstrap.min.js"></script>
</head>
<body><div id='hiddenLocation' style='display:none;'><span id='ss_webid'>161</span><span id='col1_name'>Research</span><span id='col1_url'>/col/col48302/index.html</span></div>
 <div id="old-ie">您的浏览器版本太低，请使用IE9（或以上）、谷歌、火狐等现代浏览器。360、QQ、搜狗等浏览器请使用极速模式。</div>
 <header class="headbar">
  <div class="container ">
    <div class="row">
      <div class="col-sm-12  col-md-12 col-xs-12 col-lg-12 ">
        <div class="logo">
          <a href="#"><img src="/site/images/6964/logo.png" class="img-responsive"></a>
        </div>
      </div>

    </div>
  </div>
</header>
<nav class="navbar navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="sr-only"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    <div class="collapse navbar-collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown"><a id="Home" href="/site/">Home</a></li>
        <li class="dropdown"><a id="Research" href="/site/col/col48302/index.html">Research</a></li>
        <li class="dropdown"><a id="Publications" href="/site/col/col48303/index.html">Publications</a></li>
        <li class="dropdown"><a id="Team" href="/site/col/col48304/index.html">Team</a></li>
        <li class="dropdown"><a id="Join us" href="/site/col/col48305/index.html">Join us</a> </li>
        <li class="dropdown"><a id="Contact" href="/site/col/col48306/index.html">Contact</a></li>
      </ul>
    </div>
  </div>
</nav>
<section class="banner_col">
  <img src="/site/images/6964/banner1.jpg" alt="" >

</section>
<section class="navigatorbg">
<div class="container">
  <div class="row">
    <div class="col-md-12">
      <div class="navigator">
        <table cellspacing="0"cellpadding="0"border="0"><tr><td><a href="/en"><span class="glyphicon glyphicon-home home"></span>Home</a></td><td>>><a href="/site/col/col48302/index.html"class="bt_link">Research</a></td></tr></table>
<script language="javascript">
    $(document).ready(function(){
     $(".infocont").addClass('piclist');
 });
</script>

      </div>  
    </div>
  </div>
</div>
</section>
<section class="">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <!-- 3 -->
        <div class="colbox2">
              <div class="column_right">
                 
                        <div class="column_tit">
                          <span class="glyphicon glyphicon-bookmark"></span>Research

                        </div>
                  
              <div class="column_list">
                <div class="infolist" style="padding-top: 3rem;">
                  <div class="artmain">
	<div class="infocont">
	<h3 class="art_t"><!--<$[标题]>begin-->Research<!--<$[标题]>end--></h3>
	<div class="art_time">
	</div>
		<!--<$[信息内容]>begin--><!--ZJEG_RSS.content.begin--><meta name="ContentStart"><p style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="color: rgb(69, 69, 69); font-weight: bold; font-style: normal; font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 24px;">Research Interests</span></p><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">•<span style="font-size: 18px; font-family: arial, helvetica, sans-serif; color: rgb(69, 69, 69); font-weight: normal; font-style: normal;">Genetics/Genomics:&nbsp; Genetic architectures underlying complex traits and diseases</span></span></div><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-size: 18px; font-family: Arial;">•</span><span style="font-size: 18px; color: rgb(69, 69, 69); font-weight: normal; font-style: normal; font-family: arial, helvetica, sans-serif;">Noncoding RNA:&nbsp; &nbsp; &nbsp; &nbsp; Biogenesis and function of (small) noncoding transcripts</span></span></div><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: Arial; font-size: 24px;">•</span><span style="color: rgb(69, 69, 69); font-weight: normal; font-style: normal; font-family: arial, helvetica, sans-serif; font-size: 18px;">Medical imaging:&nbsp; &nbsp; &nbsp; &nbsp;Segmentation of tumor and organs at risk</span></span></div><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: Arial; font-size: 24px;">•</span><span style="color: rgb(69, 69, 69); font-weight: normal; font-style: normal; font-size: 18px; font-family: arial, helvetica, sans-serif;">Disease study:&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Etiology and biomarker (e.g., cancer, neuronal and psychiatric disorders)</span></span></div><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: Arial; font-size: 24px;">•</span><span style="color: rgb(69, 69, 69); font-weight: normal; font-style: normal; font-size: 18px; font-family: arial, helvetica, sans-serif;">Big data analytics:&nbsp; &nbsp; &nbsp;Novel algorithms for massive (biological) data analysis</span></span></div><div style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: Arial; font-size: 24px;">•</span><span style="color: rgb(69, 69, 69); font-weight: normal; font-style: normal; font-size: 18px; font-family: arial, helvetica, sans-serif;">Artificial intelligence: Heuristic search, constraint satisfaction, and combinatorial optimization</span></span></div><p><br/></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 24px;">Research - Biology, Genetics and Genomics</span><span style="font-family: Arial, sans-serif; font-size: 20px;"></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The work in the Zhang lab in this broad field is mainly computational and falls into the category of Data Sciences (or Big Data). We aim to find and understand biological “stories” through building models and identifying patterns from large quantities of genome-scale biological data. In the process, we integrate the bench work of molecular biology and develop novel methods and efficient software tools along the way. In this broad area, we pursue two overarching themes,&nbsp;<strong>network systems biology</strong>&nbsp;and&nbsp;<strong>gene regulation via noncoding RNA genes</strong>. We are interested in basic biological questions (e.g., the biogenesis of small noncoding RNAs, transcriptional and post-transcriptional gene regulation) and their applications to complex diseases (e.g., Alzheimer’s disease (AD) and psoriasis) and plant stress response (e.g., rice blast and rice bacterial blight).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 20px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span></span></p><p style="margin: 0;font-size: 16px;font-family: Calibri, sans-serif;white-space: normal;"><a></a><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em><span style="font-size: 20px; background: white;">Network-based, allele-specific genome-wide association study&nbsp;(GWAS)</span></em></strong><strong><span style="font-size: 20px; background: white;">:</span>&nbsp;</strong></span><span style="background-color: rgb(255, 255, 255); font-family: arial, helvetica, sans-serif; font-size: 18px;">GWAS and single-gene approaches have been popular in identifying risk factors of Mendelian diseases, such as familial Alzheimer&#39;s disease (AD). Despite a tremendous amount of effort, however, these methods fail to decipher the missing inheritability of complex diseases, such as late-onset AD (or simply AD), diabetes, and psoriasis. A key factor that impairs conventional GWAS on polygenic diseases is the lack of effective ways to identify multiple genomic loci or genes whose interactions or associations underlie the manifestation of disease phenotypes. For example, a brute-force search of possible associations among only three markers within one million single nucleotide polymorphisms (SNPs) requires examining a prohibitive number of 1.7×10<sup>17</sup>&nbsp;trios. In the Zhang lab, we took a network perspective toward this problem and focused on developing novel enabling techniques for network GWAS. One network GWAS method recently developed in the lab is called&nbsp;<em>BlocBuster</em>&nbsp;(<a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>), which exploits the fact that there exist genetic architectures of markers underlying a complex trait. Such architectures are best represented as a network where correlated markers can be regarded as forming network modules or communities. We also used a new similarity measure to accommodate genetic heterogeneity in the population in order to make the network GWAS analysis allele-specific, robust, and rigorous. One application of the network GWAS method to the&nbsp;HapMap&nbsp;population SNPs data revealed a giant 284-SNPs divergent yin-yang haplotype pattern encompassing&nbsp;<em>gephyrin</em>, a gene with remarkably diverse functions related to translation initiation, regulation of synaptic protein synthesis, and&nbsp;mTOR&nbsp;signaling, to name a few (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25813846" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>Nature Comm.</em>, 6:6534, 2015</a>; see the first figure below), suggest a strong positive selection on the region over<em>&nbsp;gephyrin</em>, rapid evolution of world populations, and possible distinct molecular mechanisms involving&nbsp;<em>gephyrin</em>. The discovery of this large yin-yang pattern, which was missed by all existing methods, demonstrates the power of the network GWAS approach. The second application of the network method to GWAS data of psoriasis, an autoimmune skin disorder, helped reveal a module of 17 tightly correlated SNPs within the Major Histocompatibility Complex (MHC) on human chromosome 6 (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>; see also the second figure below). While some of the 17 SNPs have been indicated in previous studies, our result showed that their specific alleles as a whole provide a stronger discriminative power to distinguish psoriasis subjects from normal controls. In short, the Zhang lab is the first to systematically pursue the notion of network GWAS to push the&nbsp;<span style="font-family: arial, helvetica, sans-serif; background-color: rgb(255, 255, 255); color: windowtext;">envelopment of conventional GWAS to a new dimension and to develop an effective enabling technique to approach complex disease.</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="color: rgb(0, 0, 153); font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 18px;"><img title="1.png" style="border: none; width: 541px; height: 328.32px;" src="/site/picture/0/d36267d66cd440de82c75a6cc3909af0.png" width="541" height="328"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img title="2.png" style="border: none; width: 469.349px; height: 250px;" src="/site/picture/0/add660da800f4bf68a2dcf39e87b71bb.png" width="469" height="250"/></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><em>Network-based transcriptome analysis</em><em>:</em><strong>&nbsp;</strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Manifestation of complex diseases or traits is often the result of aberrant gene expression variations in disease states. Analysis of genome-wide gene expression variations, or transcriptome analysis, can thus provide insights into disease mechanisms. In the same vein as network GWAS, the Zhang lab has taken a co-expression network approach to transcriptome analysis of complex diseases by exploiting the rationale of “guilty by association”. One realization of this idea is the&nbsp;HQCut&nbsp;method for constructing and analyzing co-expression networks using gene expression data (<a href="https://bmcsystbiol.biomedcentral.com/track/pdf/10.1186/1752-0509-4-8" style="color: rgb(211, 1, 33);">Ruan, Dean and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>;&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18351912" style="color: rgb(211, 1, 33);">Ruan&nbsp;and Zhang,&nbsp;<em>Physics Review E</em>, 77:016104, 2008</a>). The central piece of the method is a heuristic that combines a spectral clustering technique that maps gene expression data to a latent, lower dimensional space, by which the noise of low information content in the data can be effectively removed. As a result, clustering can be more accurately and efficiently performed. This network transcriptome method has been used in our own research and by many other labs. In particular, using this method, we were able to unravel intricate relationships among co-expressed genes in AD, diabetes, and cardiovascular diseases (<a href="https://www.ncbi.nlm.nih.gov/pubmed/18842138" style="color: rgb(211, 1, 33);">Ray,&nbsp;Ruan&nbsp;and Zhang,&nbsp;<em>Genome Biol.</em>, 9(10):R148, 2008</a>), showing that many genes that are known to be involved in these metabolic diseases were also aberrantly expressed in the AD brain and more interestingly, appeared in one module of co-expressed genes (see the module and genes in yellow in the figure below). This result&nbsp;support&nbsp;the notion that AD is a &quot;diabetes&quot; of the brain. In addition, using this method we identified brain-region specific patterns of gene expression variation in the AD brain (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20925940" style="color: rgb(211, 1, 33);">Ray and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>) and revealed the genetics of gene expression (or&nbsp;eQTL) in AD (<a href="https://www.ncbi.nlm.nih.gov/pubmed/19361613" style="color: rgb(211, 1, 33);">Webster, et al.,&nbsp;<em>Am. J. Human Genetics</em>, 84(4):445-58, 2009</a>). In short, our work on network biology has clearly demonstrated the power and rigor that computational biology can bring to complex biology problems.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px;"><img title="3.png" style="border: none; width: 428.633px; height: 417px;" src="/site/picture/0/86f4c47baa5343529dc753ba5dc11fdf.png" width="428" height="417"/></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Biogenesis of small noncoding RNAs</em></strong><strong>:</strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif;"></span><span style="font-family: Arial, sans-serif;">&nbsp;</span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Gene expression variations across disease states and the normal condition are often due to aberrant gene regulation where small noncoding RNAs (sncRNAs), particularly microRNAs (miRNAs) and small interfering RNAs (siRNAs), play essential roles. Identification of&nbsp;sncRNAs&nbsp;and analysis of their biogenesis and regulatory functions are key steps toward elucidation of aberrant gene regulation. In my lab, we have developed effective methods for identification of miRNAs and siRNAs as well as methods for analyzing their biogenesis and potential functions in gene regulation and diseases. In particular, we developed an innovative method to show that most miRNAs in animals (e.g., human and mouse) and plants (e.g., rice and Arabidopsis) are transcribed by RNA polymerase II (<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030037" style="color: rgb(211, 1, 33);">Zhou,&nbsp;Ruan, Wang and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 3(3):e37, 2007</a>). The analysis was done by first finding&nbsp;<em>cis</em>-regulatory elements (motifs) in the promoter regions of miRNAs using our&nbsp;WordSpy&nbsp;genome-wide motif finding method (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2006-7-6-r49" style="color: rgb(211, 1, 33);">Wang and Zhang,&nbsp;<em>Genome Biol.</em>, 7(6):R49, 2006</a>), constructing a model of miRNA transcription using the motifs, and then applying the model to distinguish miRNAs from protein-coding genes. By combining large quantities of sequencing data with bench experiments, we also showed that a miRNA precursor might generate multiple miRNAs (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-8-r81" style="color: rgb(211, 1, 33);">Zhang, et al.,&nbsp;<em>Genome Biol.</em>, 11(8):R81, 2010</a>; see an example of miR822 in Arabidopsis in the first figure below) or a miRNA plus a siRNA that regulates DNA methylation, suggesting a model of dual biogenesis of miRNA and siRNA (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20621980" style="color: rgb(211, 1, 33);">Chellappan, et al.,&nbsp;<em>Nucleic Acids Res.</em>, 38(20):6883-94, 2010</a>; see the second figure below)</span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px;"><img title="4.png" style="border: none; width: 652.412px; height: 345px;" src="/site/picture/0/dbfa81a5d1a14ff0811f82bff0b8b74b.png" width="652" height="345"/></span><span style="font-size: 18px;"><span style="font-size: 20px; font-family: Arial, sans-serif;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="font-size: 20px; font-family: Arial, sans-serif; color: rgb(0, 0, 153);"><img title="5.png" style="border: none; width: 363.454px; height: 324px;" src="/site/picture/0/fd6c16dbdace44aba332ea931222eb9c.png" width="363" height="324"/></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">Moreover, microRNAs may also arise from the same&nbsp;gemomic&nbsp;loci as other noncoding RNAs, e.g.,&nbsp;tRNAs&nbsp;and&nbsp;snoRNAs. Combining computational and bench molecular biology work, we identified and confirmed that miRNAs can originate from the loci where&nbsp;tRNAs&nbsp;were encoded in mouse and murine herpesvirus (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20660200" style="color: rgb(211, 1, 33);">Reese, et al.,&nbsp;<em>J. Virology</em>, 84(19):1034-53, 2010</a>; see the first figure below for some examples in mouse). In addition, our extensive computational analysis of large quantities of profiling data revealed that there exist a huge number of isoforms of microRNAs in many eukaryotic organisms, including human (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24178030" style="color: rgb(211, 1, 33);">Xia and Zhang,&nbsp;<em>Nucleic Acids Res.</em>, 42(3):1427-41, 2014</a>; also see the second figure below for an example), some of which may also be functional in complex diseases such as psoriasis (<a href="https://www.ncbi.nlm.nih.gov/pubmed/23175445" style="color: rgb(211, 1, 33);">Xia, Joyce, Bowcock and Zhang,&nbsp;<em>Human Mol. Genetics</em>, 22(4):737-48, 2013</a>). In summary, our work has broadened our view on the biogenesis, diversity and function of small noncoding RNAs.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px;"><img title="6.png" style="border: none; width: 378px; height: 398.268px;" src="/site/picture/0/413c848201084781aff2c0d11083054a.png" width="378" height="398"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img title="7.png" style="border: none; width: 667px; height: 317.708px;" src="/site/picture/0/1e25245cc5e54cbeac280aeb52c53633.png" width="667" height="317"/></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Regulatory functions of small noncoding RNAs: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">We expanded our work to examine the potential roles of&nbsp;sncRNAs&nbsp;in complex disease and plant stress response. For example, using the miRNA transcriptome modeling as a tool, we were able to identify cold- and UV-B-light responsive miRNAs in model plant Arabidopsis thaliana, most of which were also experimentally validated (<a href="https://www.ncbi.nlm.nih.gov/pubmed/17437028" style="color: rgb(211, 1, 33);">Zhou, Wang and Zhang,&nbsp;<em>Mol. Syst. Biol.</em>, 3:13, 2007</a>;&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18471443" style="color: rgb(211, 1, 33);">Zhou, et al.,&nbsp;<em>Biochim</em><em>.&nbsp;Biophys.&nbsp;Acta,</em>&nbsp;1779(11):780-8, 2008</a>). This is the first computational miRNA gene function annotation method by modeling the possible transcriptional regulators of microRNAs and their potential targets (see the first figure below for an example of putative Auxin signaling pathways mediated by microRNAs in response to UV-B light stress). Furthermore, combining deep-sequencing profiling, computational analysis, and experimental validation, we characterized the functions of microRNAs in psoriatic and normal human skin (<a href="https://www.ncbi.nlm.nih.gov/pubmed/21807764" style="color: rgb(211, 1, 33);">Joyce and et al.,&nbsp;<em>Human Mol. Genetics</em>, 20(20):4025-40, 2011</a>; see some microRNA aberrant expression patterns in the second figure below and microR-135b expressed in psoriatic skin in the third figure below), in viral infection (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20660200" style="color: rgb(211, 1, 33);">Reese, et al.,&nbsp;<em>J. Virology</em>, 84(19):1034-53, 2010</a>), and plant stress responses, for example, in&nbsp;Aribidopsis&nbsp;(<a href="https://www.ncbi.nlm.nih.gov/pubmed/21153682" style="color: rgb(211, 1, 33);">Zhang, et al.,&nbsp;<em>Human Mol. Genet.</em>, 20:4025-40, 2011</a>) and cassava (<a href="https://www.ncbi.nlm.nih.gov/pubmed/19942686" style="color: rgb(211, 1, 33);">Zeng, et al.,&nbsp;<em>Nucleic Acids Res.</em>, 38(3):981-95, 2010</a>. Our work has broadened our view of the biogenesis, diversity and functions of&nbsp;sncRNAs&nbsp;in diseases and plants. In summary, we have done some pioneering work in developing and using computational methods to study basic biological problems related to small noncoding RNA gene regulation that were traditionally approached through bench work.</span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px;"><img title="8.png" style="border: none; width: 403px; height: 282.192px;" src="/site/picture/0/12bc9c67ff0246cbb83a0c842fb7bf29.png" width="403" height="282"/></span><span style="font-family: Arial, sans-serif; font-size: 18px;">&nbsp;&nbsp;&nbsp;<img title="9.png" style="border: none; width: 324.855px; height: 257px;" src="/site/picture/0/415cbb8803364f1ca5ce7ae768130a40.png" width="324" height="257"/>&nbsp;&nbsp;<img title="10.png" style="border: none; width: 309px; height: 171.572px;" src="/site/picture/0/540dd0b9f44c475781fc59faac36b98b.png" width="309" height="171"/></span></p><p style="text-align: left; margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255);"><span style="font-family: Arial, sans-serif; font-size: 18px;">&nbsp;</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-size: 24px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><span style="font-size: 24px;">Research - Machine learning/datamining</span></strong></span><span style="font-size: 20px;"><strong><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span></strong><strong><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span></strong></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">In recent years, in order to support the research activities in computational biology and genomics, we have been focusing on developing effective computational methods for finding and analyzing modular structures in large (biological) networks.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Identification of network modules in complex networks: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong></strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The network GWAS and co-expression network approaches described above rely heavily on accurately and efficiently finding network modules. Nevertheless, finding structures in large networks is a challenging problem in machine learning and datamining. First, network structural properties can be characterized in several different ways, by node connectivity, by properties on links (such as relationships among connected individuals), or by semantics of nodes and links (such as the social roles that individuals play in a society network). In our research, we considered these different types of information for network module finding and developed effective novel methods for finding modules of nodes (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/nodeDegreeAAA15.pdf" style="color: rgb(211, 1, 33);">Jin, Chen, He and Zhang,&nbsp;<em>Proc. AAAI-15</em></a>), modules of links (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);">He, Liu,&nbsp;Jin&nbsp;and Zhang,&nbsp;</a><a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);"><em><span style="font-family: arial, helvetica, sans-serif; color: black;">Proc. AAAI-15</span></em></a>), and modules that are defined by network structural information and node semantics (<a href="https://www.cse.wustl.edu/%7Ezhang/JointIdentification.pdf" style="color: rgb(211, 1, 33);">He, et al.,&nbsp;<em>Proc</em><em>&nbsp;AAAI-17</em></a>). We adopted a few different machine learning techniques in developing our methods, including non-negative matrix factorization (Wang, et al.,&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/aaai16-attributenetworks.pdf" style="color: rgb(211, 1, 33);"><em>Proc. AAAI-16</em></a>), stochastic modeling (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);">He, Liu,&nbsp;Jin&nbsp;and Zhang,&nbsp;</a><a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);"><em><span style="font-family: arial, helvetica, sans-serif; color: black;">Proc. AAAI-15</span></em></a>), deep learning (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/ijcai16-DL.pdf" style="color: rgb(211, 1, 33);">Yang, et al.,&nbsp;<em>Proc. IJCAI-16</em></a>), and Markov Random Fields (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MRF-aaai18.pdf" style="color: rgb(211, 1, 33);">He, et al.,&nbsp;<em>Proc. AAAI-18</em></a>).</span></span></p><p style="margin: 	</div>
	    
	<div class="share">
	  
	  <a class="close1" href="javascript:window.opener=null;window.open('','_self');window.close();">close</a>
		<a class="print" href="javascript:window.print()">print</a>
	</div></div>
 
                </div>
                
                  

                </div>      
              </div>
      </div>
      
      </div>
    </div>
  </div>
</section>
<footer id="footer">
<div class="container">
  <div class="row">
    <div class="col-md-12">
      <ul>
        <li>Contact: The Hong Kong Polytechnic University Lee Shau Kee Building, BlockY 922,
          Yuk Choi Rd, Hung Hom, Hong Kong
           </li>

       
      </ul>
    </div>
  </div>
</div>
</footer>
<script type="text/javascript" src="/site/images/6964/common.js"></script>


</body>
</html>
