<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"><script language="javascript" src="/module/jslib/jquery/jquery.js"></script>

<meta name='WebId' content='161'>

<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<title>Hong Kong Polytechnic University Research</title>
<meta name='Maketime' content='2023-04-13 19:42:47'>
<meta name="SiteName" content="Hong Kong Polytechnic University">
<meta name="SiteDomain" content="https://genomicmedicine.github.io/site/">
<meta name="SiteIDCode" content="">
<meta name="ColumnName" content="Research">
<meta name="ColumnDescription" content="">
<meta name="ColumnKeywords" content="">
<meta name="ColumnType" content="">
<meta name="channel" content="Research">
<meta name="chnlcatalog" content="">
<meta name="server" content="/site/col/col48302/index.html">
<meta name="i_webid" content="161">
<meta name="i_columnid" content="48302">
<meta name="i_articleid" content="0">
<meta name="pagetype" content="2">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="stylesheet" href="/site/images/6964/font-awesome.min.css">
<link href="/site/script/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="/site/images/6964/encommon.css">
<!--[if lt IE 9]>
<script src="https://cdn.bootcss.com/html5shiv/3.7.3/html5shiv.js"></script>
<script src="https://cdn.bootcss.com/respond.js/1.4.2/respond.min.js"></script>
<script>
        $(function () {
            $("#old-ie").css("display", "block");
        });
    </script>

<![endif]-->
<script src="/site/script/jquery.min.js"></script>
<script src="/site/script/bootstrap.min.js"></script>
</head>
<body><div id='hiddenLocation' style='display:none;'><span id='ss_webid'>161</span><span id='col1_name'>Research</span><span id='col1_url'>/col/col48302/index.html</span></div>
 <div id="old-ie">您的浏览器版本太低，请使用IE9（或以上）、谷歌、火狐等现代浏览器。360、QQ、搜狗等浏览器请使用极速模式。</div>
 <header class="headbar">
  <div class="container ">
    <div class="row">
      <div class="col-sm-12  col-md-12 col-xs-12 col-lg-12 ">
        <div class="logo">
          <a href="#"><img src="/site/images/6964/logo.png" class="img-responsive"></a>
        </div>
      </div>

    </div>
  </div>
</header>
<nav class="navbar navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="sr-only"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    <div class="collapse navbar-collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown"><a id="Home" href="/site/">Home</a></li>
        <li class="dropdown"><a id="Research" href="/site/col/col48302/index.html">Research</a></li>
        <li class="dropdown"><a id="Publications" href="/site/col/col48303/index.html">Publications</a></li>
        <li class="dropdown"><a id="Team" href="/site/col/col48304/index.html">Team</a></li>
        <li class="dropdown"><a id="Join us" href="/site/col/col48305/index.html">Join us</a> </li>
        <li class="dropdown"><a id="Contact" href="/site/col/col48306/index.html">Contact</a></li>
      </ul>
    </div>
  </div>
</nav>
<section class="banner_col">
  <img src="/site/images/6964/banner1.jpg" alt="" >

</section>
<section class="navigatorbg">
<div class="container">
  <div class="row">
   <!-- <div class="col-md-12">
      <div class="navigator">
        <table cellspacing="0"cellpadding="0"border="0"><tr><td><a href="/site/"><span class="glyphicon glyphicon-home home"></span>Home</a></td><td>>><a href="/site/col/col48302/index.html"class="bt_link">Research</a></td></tr></table>
<script language="javascript">
    $(document).ready(function(){
     $(".infocont").addClass('piclist');
 });
</script>

      </div>  
    </div> -->
  </div>
</div>
</section>
<section class="">
  <div class="container">
    <div class="row">
      <div class="col-md-12">
        <!-- 3 -->
        <div class="colbox2">
              <div class="column_right">
                 
              <div class="column_list">
                  <div class="artmain">
	<div class="infocont">

	 
		<!--<$[信息内容]>begin--><!--ZJEG_RSS.content.begin--><meta name="ContentStart"><p style="margin-top:0;margin-bottom:0;text-align:left;direction:ltr;unicode-bidi:embed;"><span style="color: rgb(69, 69, 69); font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 24px; font-weight: 700;">The research projects in the Lab are funded by</span></p><ul class=" list-paddingleft-2" style="width: 868.777px; text-wrap: wrap;"><li><div style="margin-top: 0px; margin-bottom: 0px; direction: ltr; unicode-bidi: embed;"><font color="#454545" face="arial, helvetica, sans-serif"><span style="font-size: 18px;">RGC - Research Grants Council, University Grants Committee, Hong Kong</span></font></div></li><li><div style="margin-top: 0px; margin-bottom: 0px; direction: ltr; unicode-bidi: embed;"><font color="#454545" face="arial, helvetica, sans-serif"><span style="font-size: 18px;">HMRF - Health and Medical Research Fund, Health Bureau, Hong Kong</span></font></div></li><li><span style="color: rgb(69, 69, 69); font-family: arial, helvetica, sans-serif; font-size: 18px; text-wrap: wrap;">NSFC - National Natural Science Foundation of China</span><br><br></li></ul><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 24px;">Research - Biology, genetics and genomics</span><span style="font-family: Arial, sans-serif; font-size: 20px;"></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; margin-bottom: 5px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The work in the Zhang lab in this broad field is mainly computational and falls into the category of Data Sciences (or Big Data). We aim to find and understand biological “stories” through building models and identifying patterns from large quantities of genome-scale biological data. In the process, we integrate the bench work of molecular biology and develop novel methods and efficient software tools along the way. In this broad area, we pursue two overarching themes,&nbsp;<strong>network systems biology</strong>&nbsp;and&nbsp;<strong>gene regulation via noncoding RNA genes</strong>. We are interested in basic biological questions (e.g., the biogenesis of small noncoding RNAs, transcriptional and post-transcriptional gene regulation) and their applications to complex diseases (e.g., Alzheimer’s disease (AD) and psoriasis) and plant stress response (e.g., rice blast and rice bacterial blight).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 20px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span></span></p><p style="margin: 0;font-size: 16px;font-family: Calibri, sans-serif;white-space: normal;"><a></a><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em><span style="font-size: 20px; background: white;">Network-based, allele-specific genome-wide association study&nbsp;(GWAS)</span></em></strong><strong><span style="font-size: 20px; background: white;">:</span>&nbsp;</strong></span><span style="background-color: rgb(255, 255, 255); font-family: arial, helvetica, sans-serif; font-size: 18px;">GWAS and single-gene approaches have been popular in identifying risk factors of Mendelian diseases, such as familial Alzheimer's disease (AD). Despite a tremendous amount of effort, however, these methods fail to decipher the missing inheritability of complex diseases, such as late-onset AD (or simply AD), diabetes, and psoriasis. A key factor that impairs conventional GWAS on polygenic diseases is the lack of effective ways to identify multiple genomic loci or genes whose interactions or associations underlie the manifestation of disease phenotypes. For example, a brute-force search of possible associations among only three markers within one million single nucleotide polymorphisms (SNPs) requires examining a prohibitive number of 1.7×10<sup>17</sup>&nbsp;trios. In the Zhang lab, we took a network perspective toward this problem and focused on developing novel enabling techniques for network GWAS. One network GWAS method recently developed in the lab is called&nbsp;<em>BlocBuster</em>&nbsp;(<a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>), which exploits the fact that there exist genetic architectures of markers underlying a complex trait. Such architectures are best represented as a network where correlated markers can be regarded as forming network modules or communities. We also used a new similarity measure to accommodate genetic heterogeneity in the population in order to make the network GWAS analysis allele-specific, robust, and rigorous. One application of the network GWAS method to the&nbsp;HapMap&nbsp;population SNPs data revealed a giant 284-SNPs divergent yin-yang haplotype pattern encompassing&nbsp;<em>gephyrin</em>, a gene with remarkably diverse functions related to translation initiation, regulation of synaptic protein synthesis, and&nbsp;mTOR&nbsp;signaling, to name a few (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25813846" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>Nature Comm.</em>, 6:6534, 2015</a>; see the first figure below), suggest a strong positive selection on the region over<em>&nbsp;gephyrin</em>, rapid evolution of world populations, and possible distinct molecular mechanisms involving&nbsp;<em>gephyrin</em>. The discovery of this large yin-yang pattern, which was missed by all existing methods, demonstrates the power of the network GWAS approach. The second application of the network method to GWAS data of psoriasis, an autoimmune skin disorder, helped reveal a module of 17 tightly correlated SNPs within the Major Histocompatibility Complex (MHC) on human chromosome 6 (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>; see also the second figure below). While some of the 17 SNPs have been indicated in previous studies, our result showed that their specific alleles as a whole provide a stronger discriminative power to distinguish psoriasis subjects from normal controls. In short, the Zhang lab is the first to systematically pursue the notion of network GWAS to push the&nbsp;<span style="font-family: arial, helvetica, sans-serif; background-color: rgb(255, 255, 255); color: windowtext;">envelopment of conventional GWAS to a new dimension and to develop an effective enabling technique to approach complex disease.</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="color: rgb(0, 0, 153); font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 18px; text-indent: 0px;"><img title="1.png" style="border: none; width: 541px; height: 328.32px;" src="/site/picture/0/d36267d66cd440de82c75a6cc3909af0.png" width="541" height="328">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img title="2.png" style="border: none; width: 469.349px; height: 250px;" src="/site/picture/0/add660da800f4bf68a2dcf39e87b71bb.png" width="469" height="250"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><em>Network-based transcriptome analysis</em><em>:</em><strong>&nbsp;</strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Manifestation of complex diseases or traits is often the result of aberrant gene expression variations in disease states. Analysis of genome-wide gene expression variations, or transcriptome analysis, can thus provide insights into disease mechanisms. In the same vein as network GWAS, the Zhang lab has taken a co-expression network approach to transcriptome analysis of complex diseases by exploiting the rationale of “guilty by association”. One realization of this idea is the&nbsp;HQCut&nbsp;method for constructing and analyzing co-expression networks using gene expression data (<a href="https://bmcsystbiol.biomedcentral.com/track/pdf/10.1186/1752-0509-4-8" style="color: rgb(211, 1, 33);">Ruan, Dean and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>;&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18351912" style="color: rgb(211, 1, 33);">Ruan&nbsp;and Zhang,&nbsp;<em>Physics Review E</em>, 77:016104, 2008</a>). The central piece of the method is a heuristic that combines a spectral clustering technique that maps gene expression data to a latent, lower dimensional space, by which the noise of low information content in the data can be effectively removed. As a result, clustering can be more accurately and efficiently performed. This network transcriptome method has been used in our own research and by many other labs. In particular, using this method, we were able to unravel intricate relationships among co-expressed genes in AD, diabetes, and cardiovascular diseases (<a href="https://www.ncbi.nlm.nih.gov/pubmed/18842138" style="color: rgb(211, 1, 33);">Ray,&nbsp;Ruan&nbsp;and Zhang,&nbsp;<em>Genome Biol.</em>, 9(10):R148, 2008</a>), showing that many genes that are known to be involved in these metabolic diseases were also aberrantly expressed in the AD brain and more interestingly, appeared in one module of co-expressed genes (see the module and genes in yellow in the figure below). This result&nbsp;support&nbsp;the notion that AD is a "diabetes" of the brain. In addition, using this method we identified brain-region specific patterns of gene expression variation in the AD brain (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20925940" style="color: rgb(211, 1, 33);">Ray and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>) and revealed the genetics of gene expression (or&nbsp;eQTL) in AD (<a href="https://www.ncbi.nlm.nih.gov/pubmed/19361613" style="color: rgb(211, 1, 33);">Webster, et al.,&nbsp;<em>Am. J. Human Genetics</em>, 84(4):445-58, 2009</a>). In short, our work on network biology has clearly demonstrated the power and rigor that computational biology can bring to complex biology problems.<a class="easyScholarDOI" target="_blank" href="https://sci-hub.se/10.1186/1752-0509-4-8"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22px" height="22px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">  <image id="image0" width="32" height="32" x="0" y="0" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAHdklEQVRYw+1WW2xcVxVd59zn3JnxPDwzHo89fsexYzvO02nSPBQgRMGqVEgFtEiABCriK0L8RHwgJFQJfpCgX6SFQgskqAlRcKAlhJa2zosSJ07i2LXj58Rjz4zHd9535r4OH3YSkiYQARI/WdLRlY7u2Xvdu89aewNP8AT/Z5DHeakmXMs3d25usKRApGBwzrJuc5ZtW7D0Cge9RCtqMp+YXEzGZ8v/UwKR+gaxofdTT6VQ/4mK4NsQqvaFo0GXFPTIkEUOFd20UpmSFkuk04uLySlWTAw5iuPnZ6+9P2Waxn9HYNOuAy2qa+PzXFV0394trU3PbG/x9TT7eb9bgihQEAIwBhimDTVfNkdm0tpbF6dTf7o4OppPTJ11ZYdOT1wdjP9HBNbv/lxvQtn4ze2be/YeOrgx3NdZI4s8BRjAAICxleedIISAEMAwLPP61FL55RNX4mfOD58LGRPHbg8NvJ/JqPajCHAPbnQ//Wx3yr31Wy/079j30te213Y2Vou5oo5c0UDZsHBpLImiZoIQAqfMgxDAshksm4HjKI0EXeKeDfWetMY3XIrRtWs61rmaq7nY3OxM8d8SaO3aUq36dx36woGnP/Pdr/QF/W4HDwDzSyUIPEWVIuDCaBK1fgVhv4KF5RKyRQNT8Rxup4sIemRwlECRBdoR9Urnx3OhyTTX1tvT3bB3Q21mbOyj+YpuPJqAs/PZg90bt77wgxd3NgQ8Dt5mDIQQpLJlKBIPr0uGIvHIlXQ011YhldVwYTQFReZBQOB3S5BFDowBHqdEP4oVxOuxoi9dFqLPP7O789Nbwk41OTc3u5Ap3cnJ3/369Tsjmrtx/9f7u2sjASdv2ytVZmCwbQaeoyiVDUwv5hDyOhBL5mFZDC6Zh5qvwCFx4DmKO5eDEgKRpyCE8gWDBGMFx1NfPfiNSMRV7Gg9dvTnr52J37iPgOnr3LZuTUPnng1RN2P3rphtAy4HD5eDR7FsQhQ4ZIo6qhQB7fUetEaq7v1OSsAAUEqQUDUMTS+DEsBmhLcYdQUCgfbN+1/0Sbd/5w14ml4fuMaP8wDQtrbbkxfD27Z1RcPVbpm3VwlQQgAKREMuMAZ4XRz29tZiVQgAAIEjYFgp1R0kMxp+fHIUN2czIIRAESk2rfGDUso73b5gS9S3u784JIQa+sd4n9frbO/pW3e57O9oi/rdIABhwFKujLFYFoQSVLsleJ0iFImHwFNwlNz1AdNmqOgWskUdC6qG69Mq3rocx7UpFZbNIPAUX9zdhF1dNSvVSQ/zkjblCzqxZi1JKHwo4AnvaDFad9c7wl31Th4gYITA65IQ8Mh4d3gRl8eXkcqvuKzMc5BEHhwlsC0bmm4hV9ahFnRkCwZKFROEAG6HgLa6Kjy3swGf3dEIhyzATA5Bu/Qd09IyhmYiu2RIizwjIoMcMvfXXEPr4t9RMXvBhbaDVjWjM+pFR4MPamFFajfnMpiM5zG/VIRa1KHpFizbBqUE9QEnuqMiwn4HWiJudDX6sDbqhc9JYOdmUBk7gcrIK6aZmSmn8hgbTvr/eGqaDROPx6N8sv/z6yvBvsOH9sl7tyuDspm8DMJJoP51oNXrwXnawbnrQCQ/GOeAyQToFoFhMdgMIATgKSByNgQYoFYOdiEOM30D5sIgrIVBWLlZs1yx1UQeIxfjwWOvXao6OXRjKkMAoKOzy11pee7whk19X/rZt3cFFSPGm/PvwLp9Flb6OlBZBqgIIvlAZD+I5AcRqwDeAUJ4MGYBpgam58DKS7C1FFBOgVXygM3AALNgcAu38sF3/zxd98aRgYlBNZOz7+sFG/d9eafq2fa9Vw/3b93TWyfbAGDqsEtx2Ooo7PQV2Ms3YWcnwbQEoGfBTA2wjRXts9VoBADlQTgHiOSF5YiYKdasjmQip0/8ZeInR48P3Pxn87vrA7NX3r4o9gT/cOT3wzW9rdUtHqfIM8qDuptAq5qAhgOAbYKZRbBKBqioYBUVTM8BZgnMNgFCQDgJENygsh+m4McrZxbLA+cmh0OlK7868UDy+6xYKxXtgGJPj2ccIU5y1fd1hJ08RykYWxU9W00gg0heECUM6mkG9XWAVveAC/SCC6wH9XeB87aBuurw5oV04YdHhyaU4q03Lpx+dUDX9X/djDKp+WK1S5z5cA5BTlRqetuCTlHgKGMPHlslxD6+CADTtMzj701kX/rFBzfF3MTruZFTv0kmFh86oXysHecSU8tehU6cu1VxxjOWp7XO6/C5ZcpxlILhkaCEgIGZiXRBf/nk1cSPfv3eIEtd/eny8PE347dj+qPOcQ/bzCWmln1C8eq1mFY4O5zi1bxB3Q6ROh08EXgOhMAG7i2tbOhTcTV//K/jqe//8oMbb79z8ZSy/OGRyfO/PaeVio8cRu5TwcMgiBJq1u5sLbvatlcF6je1N0Ua2xuD1SGfwimyIFQqppHKlqxbsXRmfHZxPp2I35BL0+cLc38bUZcSJh4DjzUVA4AvVC+J3mjQEjx+RiWFEULAGCN2pcQZOdXMxZeWF2e0x433BE9wB/8Aa3ZiVq1gDAMAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjItMTItMDFUMTQ6NDQ6MDIrMDE6MDBW0C0AAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIyLTEyLTAxVDE0OjQ0OjAyKzAxOjAwJ42VvAAAAABJRU5ErkJggg=="></image></svg></a></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="3.png" style="border: none; width: 428.633px; height: 417px;" src="/site/picture/0/86f4c47baa5343529dc753ba5dc11fdf.png" width="428" height="417"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Biogenesis of small noncoding RNAs</em></strong><strong>:</strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif;"></span><span style="font-family: Arial, sans-serif;">&nbsp;</span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Gene expression variations across disease states and the normal condition are often due to aberrant gene regulation where small noncoding RNAs (sncRNAs), particularly microRNAs (miRNAs) and small interfering RNAs (siRNAs), play essential roles. Identification of&nbsp;sncRNAs&nbsp;and analysis of their biogenesis and regulatory functions are key steps toward elucidation of aberrant gene regulation. In my lab, we have developed effective methods for identification of miRNAs and siRNAs as well as methods for analyzing their biogenesis and potential functions in gene regulation and diseases. In particular, we developed an innovative method to show that most miRNAs in animals (e.g., human and mouse) and plants (e.g., rice and Arabidopsis) are transcribed by RNA polymerase II (<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030037" style="color: rgb(211, 1, 33);">Zhou,&nbsp;Ruan, Wang and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 3(3):e37, 2007</a>). The analysis was done by first finding&nbsp;<em>cis</em>-regulatory elements (motifs) in the promoter regions of miRNAs using our&nbsp;WordSpy&nbsp;genome-wide motif finding method (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2006-7-6-r49" style="color: rgb(211, 1, 33);">Wang and Zhang,&nbsp;<em>Genome Biol.</em>, 7(6):R49, 2006</a>), constructing a model of miRNA transcription using the motifs, and then applying the model to distinguish miRNAs from protein-coding genes. By combining large quantities of sequencing data with bench experiments, we also showed that a miRNA precursor might generate multiple miRNAs (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-8-r81" style="color: rgb(211, 1, 33);">Zhang, et al.,&nbsp;<em>Genome Biol.</em>, 11(8):R81, 2010</a>; see an example of miR822 in Arabidopsis in the first figure below) or a miRNA plus a siRNA that regulates DNA methylation, suggesting a model of dual biogenesis of miRNA and siRNA (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20621980" style="color: rgb(211, 1, 33);">Chellappan, et al.,&nbsp;<em>Nucleic Acids Res.</em>, 38(20):6883-94, 2010</a>; see the second figure below)<a class="easyScholarDOI" target="_blank" href="https://sci-hub.se/10.1371/journal.pcbi.0030037"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22px" height="22px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">  <image id="image0" width="32" height="32" x="0" y="0" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAHdklEQVRYw+1WW2xcVxVd59zn3JnxPDwzHo89fsexYzvO02nSPBQgRMGqVEgFtEiABCriK0L8RHwgJFQJfpCgX6SFQgskqAlRcKAlhJa2zosSJ07i2LXj58Rjz4zHd9535r4OH3YSkiYQARI/WdLRlY7u2Xvdu89aewNP8AT/Z5DHeakmXMs3d25usKRApGBwzrJuc5ZtW7D0Cge9RCtqMp+YXEzGZ8v/UwKR+gaxofdTT6VQ/4mK4NsQqvaFo0GXFPTIkEUOFd20UpmSFkuk04uLySlWTAw5iuPnZ6+9P2Waxn9HYNOuAy2qa+PzXFV0394trU3PbG/x9TT7eb9bgihQEAIwBhimDTVfNkdm0tpbF6dTf7o4OppPTJ11ZYdOT1wdjP9HBNbv/lxvQtn4ze2be/YeOrgx3NdZI4s8BRjAAICxleedIISAEMAwLPP61FL55RNX4mfOD58LGRPHbg8NvJ/JqPajCHAPbnQ//Wx3yr31Wy/079j30te213Y2Vou5oo5c0UDZsHBpLImiZoIQAqfMgxDAshksm4HjKI0EXeKeDfWetMY3XIrRtWs61rmaq7nY3OxM8d8SaO3aUq36dx36woGnP/Pdr/QF/W4HDwDzSyUIPEWVIuDCaBK1fgVhv4KF5RKyRQNT8Rxup4sIemRwlECRBdoR9Urnx3OhyTTX1tvT3bB3Q21mbOyj+YpuPJqAs/PZg90bt77wgxd3NgQ8Dt5mDIQQpLJlKBIPr0uGIvHIlXQ011YhldVwYTQFReZBQOB3S5BFDowBHqdEP4oVxOuxoi9dFqLPP7O789Nbwk41OTc3u5Ap3cnJ3/369Tsjmrtx/9f7u2sjASdv2ytVZmCwbQaeoyiVDUwv5hDyOhBL5mFZDC6Zh5qvwCFx4DmKO5eDEgKRpyCE8gWDBGMFx1NfPfiNSMRV7Gg9dvTnr52J37iPgOnr3LZuTUPnng1RN2P3rphtAy4HD5eDR7FsQhQ4ZIo6qhQB7fUetEaq7v1OSsAAUEqQUDUMTS+DEsBmhLcYdQUCgfbN+1/0Sbd/5w14ml4fuMaP8wDQtrbbkxfD27Z1RcPVbpm3VwlQQgAKREMuMAZ4XRz29tZiVQgAAIEjYFgp1R0kMxp+fHIUN2czIIRAESk2rfGDUso73b5gS9S3u784JIQa+sd4n9frbO/pW3e57O9oi/rdIABhwFKujLFYFoQSVLsleJ0iFImHwFNwlNz1AdNmqOgWskUdC6qG69Mq3rocx7UpFZbNIPAUX9zdhF1dNSvVSQ/zkjblCzqxZi1JKHwo4AnvaDFad9c7wl31Th4gYITA65IQ8Mh4d3gRl8eXkcqvuKzMc5BEHhwlsC0bmm4hV9ahFnRkCwZKFROEAG6HgLa6Kjy3swGf3dEIhyzATA5Bu/Qd09IyhmYiu2RIizwjIoMcMvfXXEPr4t9RMXvBhbaDVjWjM+pFR4MPamFFajfnMpiM5zG/VIRa1KHpFizbBqUE9QEnuqMiwn4HWiJudDX6sDbqhc9JYOdmUBk7gcrIK6aZmSmn8hgbTvr/eGqaDROPx6N8sv/z6yvBvsOH9sl7tyuDspm8DMJJoP51oNXrwXnawbnrQCQ/GOeAyQToFoFhMdgMIATgKSByNgQYoFYOdiEOM30D5sIgrIVBWLlZs1yx1UQeIxfjwWOvXao6OXRjKkMAoKOzy11pee7whk19X/rZt3cFFSPGm/PvwLp9Flb6OlBZBqgIIvlAZD+I5AcRqwDeAUJ4MGYBpgam58DKS7C1FFBOgVXygM3AALNgcAu38sF3/zxd98aRgYlBNZOz7+sFG/d9eafq2fa9Vw/3b93TWyfbAGDqsEtx2Ooo7PQV2Ms3YWcnwbQEoGfBTA2wjRXts9VoBADlQTgHiOSF5YiYKdasjmQip0/8ZeInR48P3Pxn87vrA7NX3r4o9gT/cOT3wzW9rdUtHqfIM8qDuptAq5qAhgOAbYKZRbBKBqioYBUVTM8BZgnMNgFCQDgJENygsh+m4McrZxbLA+cmh0OlK7868UDy+6xYKxXtgGJPj2ccIU5y1fd1hJ08RykYWxU9W00gg0heECUM6mkG9XWAVveAC/SCC6wH9XeB87aBuurw5oV04YdHhyaU4q03Lpx+dUDX9X/djDKp+WK1S5z5cA5BTlRqetuCTlHgKGMPHlslxD6+CADTtMzj701kX/rFBzfF3MTruZFTv0kmFh86oXysHecSU8tehU6cu1VxxjOWp7XO6/C5ZcpxlILhkaCEgIGZiXRBf/nk1cSPfv3eIEtd/eny8PE347dj+qPOcQ/bzCWmln1C8eq1mFY4O5zi1bxB3Q6ROh08EXgOhMAG7i2tbOhTcTV//K/jqe//8oMbb79z8ZSy/OGRyfO/PaeVio8cRu5TwcMgiBJq1u5sLbvatlcF6je1N0Ua2xuD1SGfwimyIFQqppHKlqxbsXRmfHZxPp2I35BL0+cLc38bUZcSJh4DjzUVA4AvVC+J3mjQEjx+RiWFEULAGCN2pcQZOdXMxZeWF2e0x433BE9wB/8Aa3ZiVq1gDAMAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjItMTItMDFUMTQ6NDQ6MDIrMDE6MDBW0C0AAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIyLTEyLTAxVDE0OjQ0OjAyKzAxOjAwJ42VvAAAAABJRU5ErkJggg=="></image></svg></a></span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="4.png" style="border: none; width: 652.412px; height: 345px;" src="/site/picture/0/dbfa81a5d1a14ff0811f82bff0b8b74b.png" width="652" height="345"></span><span style="font-size: 18px; text-indent: 0px;"><span style="font-size: 20px; font-family: Arial, sans-serif;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span style="font-size: 20px; font-family: Arial, sans-serif; color: rgb(0, 0, 153); text-indent: 0px;"><img title="5.png" style="border: none; width: 363.454px; height: 324px;" src="/site/picture/0/fd6c16dbdace44aba332ea931222eb9c.png" width="363" height="324"></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">Moreover, microRNAs may also arise from the same&nbsp;gemomic&nbsp;loci as other noncoding RNAs, e.g.,&nbsp;tRNAs&nbsp;and&nbsp;snoRNAs. Combining computational and bench molecular biology work, we identified and confirmed that miRNAs can originate from the loci where&nbsp;tRNAs&nbsp;were encoded in mouse and murine herpesvirus (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20660200" style="color: rgb(211, 1, 33);">Reese, et al.,&nbsp;<em>J. Virology</em>, 84(19):1034-53, 2010</a>; see the first figure below for some examples in mouse). In addition, our extensive computational analysis of large quantities of profiling data revealed that there exist a huge number of isoforms of microRNAs in many eukaryotic organisms, including human (<a href="https://www.ncbi.nlm.nih.gov/pubmed/24178030" style="color: rgb(211, 1, 33);">Xia and Zhang,&nbsp;<em>Nucleic Acids Res.</em>, 42(3):1427-41, 2014</a>; also see the second figure below for an example), some of which may also be functional in complex diseases such as psoriasis (<a href="https://www.ncbi.nlm.nih.gov/pubmed/23175445" style="color: rgb(211, 1, 33);">Xia, Joyce, Bowcock and Zhang,&nbsp;<em>Human Mol. Genetics</em>, 22(4):737-48, 2013</a>). In summary, our work has broadened our view on the biogenesis, diversity and function of small noncoding RNAs.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="6.png" style="border: none; width: 378px; height: 398.268px;" src="/site/picture/0/413c848201084781aff2c0d11083054a.png" width="378" height="398">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img title="7.png" style="border: none; width: 667px; height: 317.708px;" src="/site/picture/0/1e25245cc5e54cbeac280aeb52c53633.png" width="667" height="317"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Regulatory functions of small noncoding RNAs: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">We expanded our work to examine the potential roles of&nbsp;sncRNAs&nbsp;in complex disease and plant stress response. For example, using the miRNA transcriptome modeling as a tool, we were able to identify cold- and UV-B-light responsive miRNAs in model plant Arabidopsis thaliana, most of which were also experimentally validated (<a href="https://www.ncbi.nlm.nih.gov/pubmed/17437028" style="color: rgb(211, 1, 33);">Zhou, Wang and Zhang,&nbsp;<em>Mol. Syst. Biol.</em>, 3:13, 2007</a>;&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18471443" style="color: rgb(211, 1, 33);">Zhou, et al.,&nbsp;<em>Biochim</em><em>.&nbsp;Biophys.&nbsp;Acta,</em>&nbsp;1779(11):780-8, 2008</a>). This is the first computational miRNA gene function annotation method by modeling the possible transcriptional regulators of microRNAs and their potential targets (see the first figure below for an example of putative Auxin signaling pathways mediated by microRNAs in response to UV-B light stress). Furthermore, combining deep-sequencing profiling, computational analysis, and experimental validation, we characterized the functions of microRNAs in psoriatic and normal human skin (<a href="https://www.ncbi.nlm.nih.gov/pubmed/21807764" style="color: rgb(211, 1, 33);">Joyce and et al.,&nbsp;<em>Human Mol. Genetics</em>, 20(20):4025-40, 2011</a>; see some microRNA aberrant expression patterns in the second figure below and microR-135b expressed in psoriatic skin in the third figure below), in viral infection (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20660200" style="color: rgb(211, 1, 33);">Reese, et al.,&nbsp;<em>J. Virology</em>, 84(19):1034-53, 2010</a>), and plant stress responses, for example, in&nbsp;Aribidopsis&nbsp;(<a href="https://www.ncbi.nlm.nih.gov/pubmed/21153682" style="color: rgb(211, 1, 33);">Zhang, et al.,&nbsp;<em>Human Mol. Genet.</em>, 20:4025-40, 2011</a>) and cassava (<a href="https://www.ncbi.nlm.nih.gov/pubmed/19942686" style="color: rgb(211, 1, 33);">Zeng, et al.,&nbsp;<em>Nucleic Acids Res.</em>, 38(3):981-95, 2010</a>. Our work has broadened our view of the biogenesis, diversity and functions of&nbsp;sncRNAs&nbsp;in diseases and plants. In summary, we have done some pioneering work in developing and using computational methods to study basic biological problems related to small noncoding RNA gene regulation that were traditionally approached through bench work.</span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="8.png" style="border: none; width: 403px; height: 282.192px;" src="/site/picture/0/12bc9c67ff0246cbb83a0c842fb7bf29.png" width="403" height="282"></span><span style="font-family: Arial, sans-serif; font-size: 18px; text-indent: 0px;">&nbsp;&nbsp;&nbsp;<img title="9.png" style="border: none; width: 324.855px; height: 257px;" src="/site/picture/0/415cbb8803364f1ca5ce7ae768130a40.png" width="324" height="257">&nbsp;&nbsp;<img title="10.png" style="border: none; width: 309px; height: 171.572px;" src="/site/picture/0/540dd0b9f44c475781fc59faac36b98b.png" width="309" height="171"></span></p><p style="text-align: left; margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255);"><span style="font-family: Arial, sans-serif; font-size: 18px;">&nbsp;</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-size: 24px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><span style="font-size: 24px;">Research - Machine learning/datamining</span></strong></span><span style="font-size: 20px;"><strong><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span></strong><strong><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span></strong></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">In recent years, in order to support the research activities in computational biology and genomics, we have been focusing on developing effective computational methods for finding and analyzing modular structures in large (biological) networks.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Identification of network modules in complex networks: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The network GWAS and co-expression network approaches described above rely heavily on accurately and efficiently finding network modules. Nevertheless, finding structures in large networks is a challenging problem in machine learning and datamining. First, network structural properties can be characterized in several different ways, by node connectivity, by properties on links (such as relationships among connected individuals), or by semantics of nodes and links (such as the social roles that individuals play in a society network). In our research, we considered these different types of information for network module finding and developed effective novel methods for finding modules of nodes (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/nodeDegreeAAA15.pdf" style="color: rgb(211, 1, 33);">Jin, Chen, He and Zhang,&nbsp;<em>Proc. AAAI-15</em></a>), modules of links (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);">He, Liu,&nbsp;Jin&nbsp;and Zhang,&nbsp;</a><a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);"><em><span style="font-family: arial, helvetica, sans-serif; color: black;">Proc. AAAI-15</span></em></a>), and modules that are defined by network structural information and node semantics (<a href="https://www.cse.wustl.edu/%7Ezhang/JointIdentification.pdf" style="color: rgb(211, 1, 33);">He, et al.,&nbsp;<em>Proc</em><em>&nbsp;AAAI-17</em></a>). We adopted a few different machine learning techniques in developing our methods, including non-negative matrix factorization (Wang, et al.,&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/aaai16-attributenetworks.pdf" style="color: rgb(211, 1, 33);"><em>Proc. AAAI-16</em></a>), stochastic modeling (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);">He, Liu,&nbsp;Jin&nbsp;and Zhang,&nbsp;</a><a href="https://www.cse.wustl.edu/%7Ezhang/publications/stocModelaaai15.pdf" style="color: rgb(211, 1, 33);"><em><span style="font-family: arial, helvetica, sans-serif; color: black;">Proc. AAAI-15</span></em></a>), deep learning (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/ijcai16-DL.pdf" style="color: rgb(211, 1, 33);">Yang, et al.,&nbsp;<em>Proc. IJCAI-16</em></a>), and Markov Random Fields (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MRF-aaai18.pdf" style="color: rgb(211, 1, 33);">He, et al.,&nbsp;<em>Proc. AAAI-18</em></a>).</span></span></p><p style="margin: 0in 0in 16px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em>Applications of network module analysis</em></strong><strong>:</strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 20px;"><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"></span><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;">&nbsp;</span><span style="font-size: 20px; font-family: Arial, sans-serif;">One of the applications of this line of research is the network-based GWAS, as discussed in&nbsp;</span><span style="font-family: Arial, sans-serif; font-size: 20px;"><a href="https://www.ncbi.nlm.nih.gov/pubmed/25233071" style="color: rgb(211, 1, 33);">(Climer, Templeton and Zhang,&nbsp;<em>PLOS Comp. Biol.</em>, 10(9):e1003766, 2014</a>) and (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25813846" style="color: rgb(211, 1, 33);">Climer, Templeton and Zhang,&nbsp;<em>Nature Comm.</em>, 6:6534, 2015</a>). Another area of application is network-based transcriptome analysis for elucidation of gene regulation, which has been studied in the Zhang lab, as discussed in (<a href="https://www.ncbi.nlm.nih.gov/pubmed/18842138" style="color: rgb(211, 1, 33);">Ray,&nbsp;Ruan&nbsp;and Zhang,&nbsp;<em>Genome Biol.</em>, 9(10):R148, 2008</a>),&nbsp;<a href="https://bmcsystbiol.biomedcentral.com/track/pdf/10.1186/1752-0509-4-8" style="color: rgb(211, 1, 33);">(Ruan, Dean and Zhang,&nbsp;<em>BMC Syst. Biol.</em>, 4:136, 2010</a>) and&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/18351912" style="color: rgb(211, 1, 33);">(Ruan&nbsp;and Zhang,&nbsp;<em>Physics Review E</em>, 77:016104, 2008</a>).<a class="easyScholarDOI" target="_blank" href="https://sci-hub.se/10.1186/1752-0509-4-8"><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22px" height="22px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">  <image id="image0" width="32" height="32" x="0" y="0" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAHdklEQVRYw+1WW2xcVxVd59zn3JnxPDwzHo89fsexYzvO02nSPBQgRMGqVEgFtEiABCriK0L8RHwgJFQJfpCgX6SFQgskqAlRcKAlhJa2zosSJ07i2LXj58Rjz4zHd9535r4OH3YSkiYQARI/WdLRlY7u2Xvdu89aewNP8AT/Z5DHeakmXMs3d25usKRApGBwzrJuc5ZtW7D0Cge9RCtqMp+YXEzGZ8v/UwKR+gaxofdTT6VQ/4mK4NsQqvaFo0GXFPTIkEUOFd20UpmSFkuk04uLySlWTAw5iuPnZ6+9P2Waxn9HYNOuAy2qa+PzXFV0394trU3PbG/x9TT7eb9bgihQEAIwBhimDTVfNkdm0tpbF6dTf7o4OppPTJ11ZYdOT1wdjP9HBNbv/lxvQtn4ze2be/YeOrgx3NdZI4s8BRjAAICxleedIISAEMAwLPP61FL55RNX4mfOD58LGRPHbg8NvJ/JqPajCHAPbnQ//Wx3yr31Wy/079j30te213Y2Vou5oo5c0UDZsHBpLImiZoIQAqfMgxDAshksm4HjKI0EXeKeDfWetMY3XIrRtWs61rmaq7nY3OxM8d8SaO3aUq36dx36woGnP/Pdr/QF/W4HDwDzSyUIPEWVIuDCaBK1fgVhv4KF5RKyRQNT8Rxup4sIemRwlECRBdoR9Urnx3OhyTTX1tvT3bB3Q21mbOyj+YpuPJqAs/PZg90bt77wgxd3NgQ8Dt5mDIQQpLJlKBIPr0uGIvHIlXQ011YhldVwYTQFReZBQOB3S5BFDowBHqdEP4oVxOuxoi9dFqLPP7O789Nbwk41OTc3u5Ap3cnJ3/369Tsjmrtx/9f7u2sjASdv2ytVZmCwbQaeoyiVDUwv5hDyOhBL5mFZDC6Zh5qvwCFx4DmKO5eDEgKRpyCE8gWDBGMFx1NfPfiNSMRV7Gg9dvTnr52J37iPgOnr3LZuTUPnng1RN2P3rphtAy4HD5eDR7FsQhQ4ZIo6qhQB7fUetEaq7v1OSsAAUEqQUDUMTS+DEsBmhLcYdQUCgfbN+1/0Sbd/5w14ml4fuMaP8wDQtrbbkxfD27Z1RcPVbpm3VwlQQgAKREMuMAZ4XRz29tZiVQgAAIEjYFgp1R0kMxp+fHIUN2czIIRAESk2rfGDUso73b5gS9S3u784JIQa+sd4n9frbO/pW3e57O9oi/rdIABhwFKujLFYFoQSVLsleJ0iFImHwFNwlNz1AdNmqOgWskUdC6qG69Mq3rocx7UpFZbNIPAUX9zdhF1dNSvVSQ/zkjblCzqxZi1JKHwo4AnvaDFad9c7wl31Th4gYITA65IQ8Mh4d3gRl8eXkcqvuKzMc5BEHhwlsC0bmm4hV9ahFnRkCwZKFROEAG6HgLa6Kjy3swGf3dEIhyzATA5Bu/Qd09IyhmYiu2RIizwjIoMcMvfXXEPr4t9RMXvBhbaDVjWjM+pFR4MPamFFajfnMpiM5zG/VIRa1KHpFizbBqUE9QEnuqMiwn4HWiJudDX6sDbqhc9JYOdmUBk7gcrIK6aZmSmn8hgbTvr/eGqaDROPx6N8sv/z6yvBvsOH9sl7tyuDspm8DMJJoP51oNXrwXnawbnrQCQ/GOeAyQToFoFhMdgMIATgKSByNgQYoFYOdiEOM30D5sIgrIVBWLlZs1yx1UQeIxfjwWOvXao6OXRjKkMAoKOzy11pee7whk19X/rZt3cFFSPGm/PvwLp9Flb6OlBZBqgIIvlAZD+I5AcRqwDeAUJ4MGYBpgam58DKS7C1FFBOgVXygM3AALNgcAu38sF3/zxd98aRgYlBNZOz7+sFG/d9eafq2fa9Vw/3b93TWyfbAGDqsEtx2Ooo7PQV2Ms3YWcnwbQEoGfBTA2wjRXts9VoBADlQTgHiOSF5YiYKdasjmQip0/8ZeInR48P3Pxn87vrA7NX3r4o9gT/cOT3wzW9rdUtHqfIM8qDuptAq5qAhgOAbYKZRbBKBqioYBUVTM8BZgnMNgFCQDgJENygsh+m4McrZxbLA+cmh0OlK7868UDy+6xYKxXtgGJPj2ccIU5y1fd1hJ08RykYWxU9W00gg0heECUM6mkG9XWAVveAC/SCC6wH9XeB87aBuurw5oV04YdHhyaU4q03Lpx+dUDX9X/djDKp+WK1S5z5cA5BTlRqetuCTlHgKGMPHlslxD6+CADTtMzj701kX/rFBzfF3MTruZFTv0kmFh86oXysHecSU8tehU6cu1VxxjOWp7XO6/C5ZcpxlILhkaCEgIGZiXRBf/nk1cSPfv3eIEtd/eny8PE347dj+qPOcQ/bzCWmln1C8eq1mFY4O5zi1bxB3Q6ROh08EXgOhMAG7i2tbOhTcTV//K/jqe//8oMbb79z8ZSy/OGRyfO/PaeVio8cRu5TwcMgiBJq1u5sLbvatlcF6je1N0Ua2xuD1SGfwimyIFQqppHKlqxbsXRmfHZxPp2I35BL0+cLc38bUZcSJh4DjzUVA4AvVC+J3mjQEjx+RiWFEULAGCN2pcQZOdXMxZeWF2e0x433BE9wB/8Aa3ZiVq1gDAMAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjItMTItMDFUMTQ6NDQ6MDIrMDE6MDBW0C0AAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIyLTEyLTAxVDE0OjQ0OjAyKzAxOjAwJ42VvAAAAABJRU5ErkJggg=="></image></svg></a></span></span></span><span style="font-size: 18px;"><span style="font-size: 16px;"><strong style="font-size: 18px;"><span style="font-size: 19px; font-family: Arial, sans-serif;"><br></span></strong></span></span></p><p style="margin: 0in 0in 16px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-size: 20px; font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><span style="font-size: 24px;">Research interests - Heuristic search, planning and optimization</span></strong></span><span style="font-size: 20px;"><span style="font-size: 20px; font-family: arial, helvetica, sans-serif;"><strong><span style="font-size: 24px;"></span></strong></span><span style="font-family: Arial, sans-serif; font-size: 18px;"><br></span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The Zhang has a long history of research on heuristic search, planning and optimization. The work has made significant contributions to the understanding of the nature and complexity of many search techniques, such as best-first search and depth-first branch-and-bound, and combinatorial optimization problems, such as the Traveling Salesman Problem and Constraint Satisfaction.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Asymptotic optimality of linear-space search algorithms:</em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em> </em></strong></span><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Heuristic search is one of the cornerstone of AI. The best-first search (BFS) algorithm, with the well-known A* algorithm as a special case, is time optimal. However, it requires space that is exponential in the search depth of the underlying search tree, which is impractical for large combinatorial optimization problems, such as the Traveling Salesman Problem (TSP). In contrast, a linear-space search (LSS) algorithm, with depth-first search and depth-first branch-and-bound (DFBnB) as special cases, runs linearly in search depth with a penalty of exploring more nodes in the search space. Due to their limited space requirement, LSS algorithms, including&nbsp;DFBnB, iterative deepening A* (IDA*), and recursive best-first search (RBFS), are the most popular techniques for large and difficult search problems in practice. Understanding the expected complexity of LSS algorithms was an important and challenging&nbsp;problem.Based&nbsp;on a random search tree model proposed by Judea Pearl and Richard Karp,&nbsp;<strong>our research established the&nbsp;<em><span style="font-family: arial, helvetica, sans-serif; text-decoration: underline;">asymptotic optimality</span>&nbsp;</em>of LSS methods</strong>, showing that their expected complexity is a constant factor of that of BFS (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/bnbaij.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhand and Korf,&nbsp;<em>Artificial Intelligence</em>, 79(2):241-92, 1995</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">). Moreover, LSS algorithms may run faster than BFS on large problems, making them the algorithms for real applications.</span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">A directly and early application of our complexity results is the explanation to anomaly of the fixed horizon search, which was observed initially by Richard&nbsp;Korf&nbsp;on sliding-tile puzzles using&nbsp;DFBnB, as illustrated below. The anomaly states that for a fixed search depth,&nbsp;DFBnB&nbsp;can search the trees with larger branching factors faster. In fact, the anomaly persists regardless which search method, e.g., ID (or IDA*) or RBFS, shown below.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; font-size: 18px; text-indent: 0px;"><img title="11.png" style="border: none; width: 365px; height: 344.722px;" src="/site/picture/0/dd7eea7dbc684d22aa404423aca5e170.png" width="365" height="344"><img title="12.png" style="border: none; width: 661px; height: 331.86px;" src="/site/picture/0/24b0a4109cf841159f9678ca1c488c97.png" width="661" height="331"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The insight gained by the complexity analysis on this search anomaly helped predict the existence of a phase transition in heuristic search problems, discussed below.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Phase transitions and structures of combinatorial search: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Our study also revealed that the expected complexity of BFS and LLS algorithms exhibits a sharp and abrupt transition from polynomial to exponential time as some critical problem features (e.g., the precision of heuristic function) shift (<a href="https://www.cse.wustl.edu/%7Ezhang/publications/bnbaij.pdf" style="color: rgb(211, 1, 33);">Zhand and Korf,&nbsp;<em>Artificial Intelligence</em>, 79(2):241-92, 1995</a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/tspaij.ps" style="color: rgb(211, 1, 33);">Zhang and&nbsp;Korf,&nbsp;<em>Artificial Intelligence</em>, 81(1-2):223-39, 1996</a>;&nbsp;<a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</a>). This is similar to phase transitions of spin glasses studied in physics. A simple example of a phase transition is water changing from liquid to solid ice when the temperature drops below the freezing point. We identified that the accuracy of heuristic information is the controlling parameter of phase transitions. On the&nbsp;Judea&amp;Karp&nbsp;incremental random search tree model, the phase transition can be summarized by the following figure.</span></span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; color: rgb(0, 0, 153); font-size: 18px; text-indent: 0px;"><img title="13.png" style="border: none; width: 402px; height: 385.048px;" src="/site/picture/0/2eac6d39e18b4de4b02f846110bb9b8e.png" width="402" height="385"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The control parameter of this phase transition is the product of the mean branching factor of the search tree and the probability if an edge cost has cost 0.</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">This analysis was extended to reveal phase transitions in combinatorial optimization problems, such as the asymmetric TSP (ATSP) (<a href="https://www.sciencedirect.com/science/article/pii/0004370295000542" style="color: rgb(211, 1, 33);">Zhang and&nbsp;Korf,&nbsp;<em>Artificial Intelligence</em>, 81(1-2):223-39, 1996</a>) and Boolean Satisfiability (<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>). For the ATSP, the control parameter is the precision of the distances between cities, which is equivalent to the number of digits used to represent distances. Intuitively, when the number of digits used is small, many distances may have a high chance to have the same value, and consequently many complete tours (visiting each and every city exactly once) may have a high chance to have the same cost as well. This also means that a large number of optimal TSP tours exist; and finding one optimal TSP tour should be relatively easy. In contrast, when no two distances between cites are the same, the number of optimal TSP tour is small, so that finding an optimal solution is difficult. This phase transition is illustrated in the figure below, where the Assignment Problem (AP) is the main computation step (heuristic) for solving the ATSP.</span></p><p style="margin: 0in 0in 0px; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left; text-indent: 0px;"><span style="font-family: Arial, sans-serif; font-size: 18px; text-indent: 0px;"><img title="14.png" style="border: none; width: 367.55px; height: 300px;" src="/site/picture/0/d808eb0cb7c74017aae8fd95733deb5a.png" width="367" height="300"></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">Interestingly, there also exist phase transitions in maximum Boolean satisfiability (max SAT), which is the optimization version of the SAT where the objective is to find a variable assignment that minimize the number of violated clauses (<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><strong><span style="font-family: Arial, sans-serif;">Note that the phase transitions that we studied in optimization problems, such as finding the minimal TSP tour and the variable assignment to minimize the number of unsatisfied constraints, are fundamentally different from the phase transitions for decision problems, such as the decision version of the TSP and the SAT. For optimization problems, the phase transitions appear as an easy-to-difficult transition, whereas for decision problems, the phase transitions exhibit an easy-to-difficult-then-to-easy pattern.</span></strong><strong><span style="font-family: Arial, sans-serif;"></span></strong></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: Arial, sans-serif; font-size: 18px;">The computational complexity and phase transition are also related to the backbone of a problem, which is a set of variables that have the same values among all solutions (<a href="https://dl.acm.org/citation.cfm?id=726965" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Proc. Intern. Conf. on Principles and Practice of Constraint programming</em>, CP-2001</a>;&nbsp;<a href="https://arxiv.org/pdf/1107.0055.pdf" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>JAIR</em>, 20:471-97, 2004</a>;&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33);">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>).</span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Problem solving by exploiting phase transitions and backbones: </em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The study of phase transitions and problem structures such as backbones has changed the way we characterize the complexity of combinatorial problems, beyond the notion of worst-case complexity, and the way we solve difficult problems.&nbsp; In particular, phase transitions can not only be used to characterize problems, but also be exploited in problem solving. We have done a&nbsp;<strong>pioneering work</strong>&nbsp;on developing new efficient algorithms for finding optimal and approximate solutions by exploiting phase transitions (<a href="https://ac.els-cdn.com/0004370295000577/1-s2.0-0004370295000577-main.pdf?_tid=8adf368a-3539-4297-975c-a6d046a35b9f&amp;acdnat=1539208483_3dee19b9b4d695046e4d089ecb64578f" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Pemberton and Zhang,&nbsp;<em>Artificial Intelligence</em>, 81:297-325, 1996</a>;&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Zhang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</a>). One of the ideas is to transform a state space that is difficult to search into one that is easy to explore. Another idea is to estimate and utilize backbone structures to guide a search. These techniques are characterized as&nbsp;<strong>backbone guided search</strong>&nbsp;(<a href="https://www.sciencedirect.com/science/article/pii/S0004370204000542" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;">Z<span style="font-family: arial, helvetica, sans-serif; color: rgb(211, 1, 33); text-decoration: underline;">hang,&nbsp;<em>Artificial Intelligence</em>, 158:1-26, 2004</span></a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/bgtsp.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 16px;">Zhang and Looks,&nbsp;<em>Proc. IJCAI-2005</em></a>).</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: arial, helvetica, sans-serif; font-size: 20px;"><strong><em><span style="font-family: Arial, sans-serif;">T</span></em></strong></span><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><span style="font-size: 20px;"><strong><em>he Zhang Algorithm for the ATSP:</em></strong></span><span style="font-size: 18px;"><strong><em> </em></strong></span></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;"><strong><em><span style="font-family: Arial, sans-serif;"></span></em></strong><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The asymmetric TSP (ATSP) is an important NP-hard problem with many real applications in the areas such as scheduling. The research largely focuses on efficient approximation algorithms. The first approximation method is the&nbsp;Kanallakis-Papadimitiou&nbsp;local search algorithm, named after its inventors, two eminent scientists in computer science and operations research. The second approach is represented by my truncated&nbsp;DFBnB&nbsp;algorithm (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/atsp-aaai93-symp.ps" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>Proc. AAAI 1993 Spring&nbsp;Symp. on AI and NP-Hard Problems</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://www.springer.com/us/book/9780387988320" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Zhang,&nbsp;<em>State-Space Search</em>, Springer, 1999</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">), which has been referred to as&nbsp;<strong>the Zhang algorithm</strong>&nbsp;(</span><a href="http://dimacs.rutgers.edu/archive/Challenges/TSP/papers/stspchap.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">J<span style="font-size: 18px; font-family: Arial, sans-serif; color: rgb(211, 1, 33);">ohnson, et al.in&nbsp;<em>The Traveling Salesman Problem and its Variations</em>, pp.445-88, 2002</span></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">).</span></span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;">Long distance mutual exclusion and propositional planning: </span></em></strong></span><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">A dominating technique in planning is&nbsp;GraphPlan&nbsp;with mutual exclusions (mutex) to represent constraints among actions and events at the same planning horizon (depth). In our work, we introduced long distance mutual exclusion (londex) to capture constraints across multiple horizons, developed an efficient algorithm to efficiently compute&nbsp;londex, and designed and implemented our new&nbsp;MaxPlan&nbsp;planner using&nbsp;londex&nbsp;(<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MaxPlan-final.pdf" style="color: rgb(211, 1, 33);">Xing, Chen and Zhang,&nbsp;<em>Proc. ICAPS-06</em></a>;&nbsp;<a href="https://www.cse.wustl.edu/%7Ezhang/publications/londexaij.pdf" style="color: rgb(211, 1, 33);">Chen, Huang, Xing and Zhang,&nbsp;<em>Artificial Intelligence</em>, 173:365-91, 2009</a>). The overall work provided a new paradigm for planning. Our&nbsp;MaxPlan&nbsp;system won&nbsp;<strong>the First Place Award of the Fifth International Planning Competition in 2006</strong>&nbsp;(<a href="https://www.cse.wustl.edu/%7Ezhang/publications/MaxPlan-final.pdf" style="color: rgb(211, 1, 33);">Xing, Chen and Zhang,&nbsp;<em>Proc. ICAPS-06</em></a>).</span></span></p><p style="margin-right: 0in; margin-left: 0in; font-size: medium; font-family: &quot;Times New Roman&quot;, serif; white-space: normal; background-color: rgb(255, 255, 255); text-align: left;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;; font-size: 20px;"><strong><em>Transition based encoding for planning:</em></strong></span><span style="font-size: 18px;"><span style="font-family: &quot;arial black&quot;, &quot;avant garde&quot;;"><strong><em> </em></strong></span><span style="font-family: Arial, sans-serif;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">The technique of planning as Boolean satisfiability usually uses encodings in logic statements. We introduced a novel SAT encoding scheme (SASE) based on the SAS+ formalism (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/SASE-Encoding-aaai10.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>Proc. AAAI-10</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">;&nbsp;</span><a href="https://jair.org/index.php/jair/article/view/10751" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>JAIR</em>, 43:293-328, 2012</span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">). The new scheme exploits the structural information so that the encoding is both compact and efficient for planning. We proved the correctness of the new encoding and analyzed the rationale behind its improvement in planning performance. Our extensive experimental results demonstrated significant improvements of SASE over the state-of-the-art methods in time and space. This work received&nbsp;<strong>the Outstanding Paper Award</strong>&nbsp;of the 24-th AAAI Conference on Artificial Intelligence Conference in 2010 (AAA-10) (</span><a href="https://www.cse.wustl.edu/%7Ezhang/publications/SASE-Encoding-aaai10.pdf" style="color: rgb(211, 1, 33); text-decoration: underline; font-family: arial, helvetica, sans-serif; font-size: 18px;"><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">Huang, Chen and Zhang,&nbsp;<em>Proc. AAAI-10</em></span></a><span style="font-family: arial, helvetica, sans-serif; font-size: 18px;">), chosen from a double blind review process of 894 submissions. Note that AAAI is one of the two leading international AI conferences.</span></span></span></p><p><br></p><meta name="ContentEnd"><!--ZJEG_RSS.content.end--><!--<$[信息内容]>end-->
	 
	<div class="share">
	  
	  <a class="close1" href="javascript:window.opener=null;window.open('','_self');window.close();">close</a>
		<a class="print" href="javascript:window.print()">print</a>
	</div></div>
 
                </div>
                
                  

                </div>      
              </div>
      </div>
      
      </div>
    </div>
  </div>
</section>
<footer id="footer">
<div class="container">
  <div class="row">
    <div class="col-md-12">
      <ul>
        <li>Contact: The Hong Kong Polytechnic University Lee Shau Kee Building, BlockY 922,
          Yuk Choi Rd, Hung Hom, Hong Kong
           </li>

       
      </ul>
    </div>
  </div>
</div>
</footer>
<script type="text/javascript" src="/site/images/6964/common.js"></script>


</body>
</html>
